assets/css/_vehicle_route_summary.scss

```diff
@@ -84,6 +84,26 @@
     background-color: $color-gray-300;
     width: 1px;
   }
+  // #endregion
+  // #endregion
```

Comment:
**question (non-blocking): is this a mistake or are there nested regions? I'm not really familiar with this syntax to be honest.